src/signed-data-store/signed-data-store.test.ts

```diff
@@ -75,4 +77,27 @@ describe('datastore', () => {
 
     expect(verifySignedDataIntegrity(badTestDataPoint)).toBeFalsy();
   });
+
+  it('purges old data from the store', () => {
+    const baseTime = 1_700_126_230_000;
+    jest.useFakeTimers().setSystemTime(baseTime);
+
+    initializeState();
+    updateState((draft) => {
+      draft.signedApiStore['0x000'] = allowPartial<SignedData>({
+        timestamp: (baseTime / 1000 - 25 * 60 * 60).toString(),
+      });
+      draft.signedApiStore['0x001'] = allowPartial<SignedData>({
+        timestamp: (baseTime / 1000 - 23 * 60 * 60).toString(),
+      });
+    });
+
+    purgeOldSignedData();
+
+    expect(Object.values(getState().signedApiStore)).toStrictEqual([
+      {
+        timestamp: '1700043430',
```

Comment:
nit: `(baseTime / 1000 - 23 * 60 * 60).toString()` ?

maybe save those values as `purgableTimestamp` and `nonPurgableTimestamp`